app/views/issues/show.html.erb

```diff
@@ -54,7 +54,7 @@
                 <%= markup(@issue.text) %>
               </div>
               <div class="author-info">
-                <span class="ml-auto">Issue Author: <%= @issue.author %></span>
+                <span class="ml-auto">Author: <%= @issue.author || 'n/a' %></span>
```

Comment:
why is 'n/a' needed now?

(same for the other files)

Reply:
I noticed that when some cards have been populated from a template the author is nil. Resulting in the view rendering only `Author:`. I added this to each file in case there were to ever be something that doesn't have an author for any reason. Alternatively, we can check for author and not render anything if there is no author (this will also require a conditional class on the content-container).

Reply:
Seems inconsistent with content blocks since it doesn't use `n/a` there.

Reply:
> Seems inconsistent with content blocks since it doesn't use n/a there.

https://github.com/secroots/dradis-pro/pull/862